From 395f2f4c85764a89b725a98b395e06d50d06bea9 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Tue, 25 Apr 2017 13:57:54 +0100
Subject: [PATCH] Disable NetworkManager integration

At this point in time, Clear Linux does not use Network Manager.
As such, we keep it disabled within GNOME Initial Setup.

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 configure.ac                              | 4 ----
 gnome-initial-setup/Makefile.am           | 1 -
 gnome-initial-setup/gnome-initial-setup.c | 2 --
 gnome-initial-setup/pages/Makefile.am     | 1 -
 4 files changed, 8 deletions(-)

diff --git a/configure.ac b/configure.ac
index ecd4b19..3000ba7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -31,8 +31,6 @@ GEOCLUE_REQUIRED_VERSION=2.3.1
 PACKAGEKIT_REQUIRED_VERSION=1.1.4
 
 PKG_CHECK_MODULES(INITIAL_SETUP,
-                  libnm >= $LIBNM_REQUIRED_VERSION
-                  libnma >= $LIBNMA_REQUIRED_VERSION
                   polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
                   accountsservice
                   gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION
@@ -122,8 +120,6 @@ ISO_CODES=iso-codes
 AC_SUBST(INITIAL_SETUP_CFLAGS)
 AC_SUBST(INITIAL_SETUP_LIBS)
 
-GNOME_COMPILE_WARNINGS([error])
-GNOME_MAINTAINER_MODE_DEFINES
 
 AC_SUBST(CFLAGS)
 AC_PATH_PROG(GLIB_COMPILE_RESOURCES, glib-compile-resources)
diff --git a/gnome-initial-setup/Makefile.am b/gnome-initial-setup/Makefile.am
index d275dda..fc10c8d 100644
--- a/gnome-initial-setup/Makefile.am
+++ b/gnome-initial-setup/Makefile.am
@@ -35,7 +35,6 @@ gnome_initial_setup_LDADD =	\
 	pages/region/libgisregion.la \
 	pages/keyboard/libgiskeyboard.la \
 	pages/eulas/libgiseulas.la \
-	pages/network/libgisnetwork.la \
 	pages/timezone/libgistimezone.la \
 	pages/privacy/libgisprivacy.la \
 	pages/goa/libgisgoa.la \
diff --git a/gnome-initial-setup/gnome-initial-setup.c b/gnome-initial-setup/gnome-initial-setup.c
index 2ec91b9..f68b233 100644
--- a/gnome-initial-setup/gnome-initial-setup.c
+++ b/gnome-initial-setup/gnome-initial-setup.c
@@ -40,7 +40,6 @@
 #include "pages/region/gis-region-page.h"
 #include "pages/keyboard/gis-keyboard-page.h"
 #include "pages/eulas/gis-eula-pages.h"
-#include "pages/network/gis-network-page.h"
 #include "pages/timezone/gis-timezone-page.h"
 #include "pages/privacy/gis-privacy-page.h"
 #include "pages/software/gis-software-page.h"
@@ -66,7 +65,6 @@ static PageData page_table[] = {
   /* PAGE (region,   FALSE), */
   PAGE (keyboard, FALSE),
   PAGE (eula,     FALSE),
-  PAGE (network,  FALSE),
   PAGE (privacy,  FALSE),
   PAGE (timezone, TRUE),
   PAGE (software, TRUE),
diff --git a/gnome-initial-setup/pages/Makefile.am b/gnome-initial-setup/pages/Makefile.am
index e5182a7..3ddd9cd 100644
--- a/gnome-initial-setup/pages/Makefile.am
+++ b/gnome-initial-setup/pages/Makefile.am
@@ -4,7 +4,6 @@ SUBDIRS = \
 	region \
 	keyboard \
 	eulas \
-	network \
 	timezone \
 	privacy \
 	goa \
-- 
2.12.2

